src/main/java/thePackmaster/cards/pinnaclepack/TheSuckamidRareSkill.java

```diff
@@ -0,0 +1,50 @@
+package thePackmaster.cards.pinnaclepack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.HealAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.EquilibriumPower;
+import thePackmaster.powers.pinnaclepack.Capacitor;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class TheSuckamidRareSkill extends AbstractPinnacleCard {
```

Comment:
*The Suckamid*
The overall design of this card has two issues: Purge and Capacitor.

The use of Purge is a problem because Purge is a mechanic that is not used on normal cards by the base game. It also has several unintuitive/buggy interactions with other effects. Instead of using Purge, this card should Exhaust. While there may be ways to recover this card from Exhaust that will be abusable with the right packs, it is still preferable to use Exhaust instead of Purge here.

Instead of the Capacitor keyword, which shares its name with a base game power and isn't necessary to keyword anyway, the text for the effect should be written out. Doing this will also make it clear that the text for this card is close to or at the text length limits, which is part of the overall issue with this pack. If the text length is over the limit, the increase to maximum hand size is what I'd suggest removing.